### PR TITLE
Added command lines to enable/disable openid from console

### PR DESCRIPTION
**Pull request by lelemm**
Added command lines to enable/disable openid from console

### CODE REVIEW

**Comment by lelemm** [comment]
Testing the config.json openid parameters, I noticed that during code review of server, code rabbit made me do a change that removes the ability to login with github. I'm reverting this change on this PR too

File: src/accounts/openid.js [hunk 1]
```diff
@@ -170,11 +170,22 @@ export async function loginWithOpenIdFinalize(body) {
...
+        state: body.state,
+      });
+    } else {
+      tokenSet = await client.grant({
+        grant_type: 'authorization_code',
+        code: body.code,
+        redirect_uri: client.redirect_uris[0],
+        code_verifier,
+      });
+    }
```
[the remark this answers is not in the document]
Comment (lelemm) [reply src/accounts/openid.js:188]: how would access_token be missing from the oauth response?

[the remark this answers is not in the document]
Comment (lelemm) [reply src/accounts/openid.js:188]: all those errors would be thrown to this catch:

} catch (err) {
    console.error('OpenID grant failed:', err);
    return { error: 'openid-grant-failed' };
  }

[the remark this answers is not in the document]
Comment (lelemm) [reply src/accounts/openid.js:188]: oauth2 should be transparent to the rest of the app, the client side does not know about oauth2, it should be treated like openid by the frontend.

suggest me a code considering what I typed in the last messages of this point

[the remark this answers is not in the document]
Comment (lelemm) [reply src/accounts/openid.js:188]: I will keep the big catch in the end since it gives me the same result.

**Comment by MikesGlitch** [comment]
I got this: 
```
Unexpected error: SqliteError: FOREIGN KEY constraint failed
    at WrappedDatabase.mutate (file:///D:/GitWork/actual-server/src/db.js:39:21)
    at disableOpenID (file:///D:/GitWork/actual-server/src/account-db.js:173:18)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///D:/GitWork/actual-server/src/scripts/disable-openid.js:24:24 {
  code: 'SQLITE_CONSTRAINT_FOREIGNKEY'
}
```

**Comment by lelemm** [comment]
> I got this when running `yarn disable-openid`:
> 
> ```
> Unexpected error: SqliteError: FOREIGN KEY constraint failed
>     at WrappedDatabase.mutate (file:///D:/GitWork/actual-server/src/db.js:39:21)
>     at disableOpenID (file:///D:/GitWork/actual-server/src/account-db.js:173:18)
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
>     at async file:///D:/GitWork/actual-server/src/scripts/disable-openid.js:24:24 {
>   code: 'SQLITE_CONSTRAINT_FOREIGNKEY'
> }
> ```

fixed

File: src/scripts/enable-openid.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      console.log('OpenID already enabled.');
+      process.exit(0);
+    }
+    const { error } = (await enableOpenID(finalConfig)) || {};
+
+    if (error) {
+      console.log('Error enabling openid:', error);
+      console.log(
+        'Please report this as an issue: https://github.com/actualbudget/actual-server/issues',
+      );
```
[the remark this answers is not in the document]
Comment (lelemm) [reply src/scripts/enable-openid.js]: changed to your suggestion

**Review by MikesGlitch — APPROVED** [review]

**Comment by MikesGlitch** [comment]
LGTM

**Pull request merged**